.github/workflows/jdk-ea-stable.yml

```diff
@@ -0,0 +1,37 @@
+
+name: avaje-jsonb EA stable
+
+on:
+  push:
+  pull_request:
+  workflow_dispatch:
+  schedule:
+    - cron: '39 1 * * 1,3,5'
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read
+      packages: write
+
+    steps:
+    - uses: actions/checkout@v3
+    - name: Set up Java
+      uses: oracle-actions/setup-java@v1
+      with:
+        website: jdk.java.net
+        release: ea
+        version: stable
+    - name: Maven cache
+      uses: actions/cache@v3
```

Comment:
Cool. Note that there are @v4 versions of the actions now like actions/cache@v4

Reply:
eh dependabot will get to it